business-central-tests/business-central-tests-gui/pom.xml

```diff
@@ -191,23 +198,16 @@
 
   <profiles>
     <profile>
-      <id>kie-drools-wb</id>
-      <properties>
-        <app.name>kie-drools-wb</app.name>
-      </properties>
-    </profile>
-
-    <profile>
-      <id>kie-wb</id>
+      <id>business-central</id>
       <properties>
-        <app.name>kie-wb</app.name>
+        <app.name>business-central</app.name>
       </properties>
     </profile>
 
     <profile>
-      <id>kie-wb-monitoring</id>
+      <id>business-monitoring</id>
       <properties>
-        <app.name>kie-wb-monitoring</app.name>
+        <app.name>business-central</app.name>
```

Comment:
the app.name should be `business-monitoring` in this profile.

Reply:
Fixed in https://github.com/kiegroup/kie-wb-distributions/pull/830/commits/6f6c1f1a27f0f24acb87880311ba67220d10271c